packages/gatsby-remark-images/package.json

```diff
@@ -23,7 +23,9 @@
     "@babel/cli": "^7.0.0",
     "@babel/core": "^7.0.0",
     "babel-preset-gatsby-package": "^0.2.2",
-    "cross-env": "^5.1.4"
+    "cross-env": "^5.1.4",
+    "hast-util-to-html": "^6.0.2",
+    "mdast-util-to-hast": "^6.0.1"
```

Comment:
Just a note - those 2 should go to `dependencies` not `devDependencies`

Reply:
no it's only used by the tests

Reply:
Ah, my bad - right - you need those to "mock" gatsby-transformer-remark passing `compiler` thingy.